app/models/manageiq/providers/autosde/storage_manager/storage_service.rb

```diff
@@ -17,25 +18,25 @@ def self.raw_create_storage_service(ext_management_system, options = {})
 
     task_id = ext_management_system.autosde_client.ServiceApi.services_post(service_to_create).task_id
     options = {
-      :target_class   => :storage_services,
+      :target_class   => nil,
       :target_id      => nil,
       :ems_id         => ext_management_system.id,
       :native_task_id => task_id,
       :interval       => 10.seconds,
-      :target_option  => "new"
+      :target_option  => "ems"
```

Comment:
Why this change?  Is the targeted refresh not working?  Seems like something we should fix not just do a full.